schedule/yast/nfs/yast2_nfs_v3__MM_client.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name:          yast2_nfs_v3__MM_client
```

Comment:
I'm not sure about the double dash `__` neither to use `MM` on the name. Perhaps we can omit that part, WDYT?

Reply:
Double dash is a typo, for MM think it makes sense as we have also some single machine nfs tests that can be easily confused

Reply:
though maybe in lowercase would be better, less Macdonaldsish

Reply:
I found it redundant, all stuff that we have is controller/remote, client/server, means that we are using MM, I would not introduce new naming. Single machine scenario could not be named only with client, as it would contain client and server, so I guess it is fine if you remove it.

Reply:
Okay, I was confused with the fact we have the module yast2_nfs_client running in single machine configuration in yast2_ncurses, for example.

Reply:
that is a test module, so I think we are safe using current way to name it.